requirements-prod.blue.txt

```diff
@@ -0,0 +1,47 @@
+asn1crypto==0.24.0
+attrs==17.4.0
+aws-encryption-sdk==1.3.3
+aws-xray-sdk==0.96
+backports.ssl-match-hostname==3.5.0.1
+boto==2.48.0
+boto3==1.6.4
+botocore==1.9.4
+certifi==2018.1.18
+cffi==1.11.5
+chardet==3.0.4
+configobj==5.0.6
+cookies==2.2.1
+credstash==1.14.0
+cryptography==2.0.0
+docker==3.1.1
+docker-pycreds==0.2.2
+docutils==0.14
+everett==0.9
+future==0.16.0
+idna==2.6
+Jinja2==2.10
+jmespath==0.9.3
+jsondiff==1.1.1
+jsonpickle==0.9.6
+jsonschema==2.6.0
+MarkupSafe==1.0
+git+git://github.com/mozilla-iam/cis.git@56d04d6b54435b7b3cb34b9b270f50e67a1b607a#egg=mozilla_iam_cis
```

Comment:
i wonder if we should start releasing, since tags are rather opaque (not  needed *right now* though)